PWGLF/Tasks/NucleiHistTask.cxx

```diff
@@ -42,7 +42,7 @@ namespace o2::aod
 namespace NucleiTableHist
 {
```

Comment:
Also here, consider in the future to split the table producer part in a task that is in the table producer and the data model accordingly. Thanks!